### PR TITLE
[persistence] Added methods to calculate variance and standard deviation for persisted items

### PR DESCRIPTION
**Pull request by cweitkamp**
- Added methods to calculate variance and standard deviation for persisted items

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/QueryablePersistenceService.java [hunk 1]
```diff
@@ -28,20 +28,19 @@
...
+     * Queries the {@link PersistenceService} for historic data with a given {@link FilterCriteria}.
      *
      * @param filter the filter to apply to the query
      * @return a time series of items
      */
     Iterable<HistoricItem> query(FilterCriteria filter);
 
     /**
-     * Returns a list of items that are stored in the persistence service
+     * Returns a set of {@link PersistenceItemInfo} about items that are stored in the persistence service. This allows
```
Comment (cweitkamp) [inline bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/QueryablePersistenceService.java:39]: Do you like a separate PR for this?

Comment (kaikreuzer) [reply bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/QueryablePersistenceService.java:39]: Might make it clearer that it is completely unrelated to this one here...

Comment (cweitkamp) [reply bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/QueryablePersistenceService.java]: Done: https://github.com/openhab/openhab-core/pull/1552

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
Please adapt the methods to also use `ZonedDateTime`, so that it is in-line with the other methods.

**Comment by cweitkamp** [comment]
> Please adapt the methods to also use ZonedDateTime, so that it is in-line with the other methods.

Done.

**Comment by kaikreuzer** [comment]
Looks as if you missed to update the itests/pomx.ml as well:
```
[ERROR]     Child module /home/travis/build/openhab/openhab-core/itests/org.openhab.core.persistence.tests of /home/travis/build/openhab/openhab-core/itests/pom.xml does not exist
```

**Review by kaikreuzer — APPROVED** [review]
Thanks!
